haml/_oisc_warning.html.haml

```diff
@@ -1,12 +1,12 @@
 -# Set defaults for parameters
-- body ||= '' # Body is optional
-- is_sticky ||= false
+- body = oisc_warning.fetch('body', '') # Body is optional
+- is_sticky = oisc_warning.fetch('is_sticky', false)
```

Comment:
When we're setting isSticky in public-website we'll always return true or false, so not sure if you still want the default 'false' here?

Reply:
Yeah I think so. These components shouldn't know too much about what public-website does, so better to say what we want the default to be explicitly.